src/libraries/System.Private.CoreLib/src/System/Reflection/SignatureType.cs

```diff
@@ -173,7 +176,10 @@ public sealed override Type MakeArrayType(int rank)
         protected sealed override bool IsCOMObjectImpl() => throw new NotSupportedException(SR.NotSupported_SignatureType);
         protected sealed override bool IsPrimitiveImpl() => throw new NotSupportedException(SR.NotSupported_SignatureType);
         public sealed override IEnumerable<CustomAttributeData> CustomAttributes => throw new NotSupportedException(SR.NotSupported_SignatureType);
+
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.Interfaces)]
         public sealed override Type[] FindInterfaces(TypeFilter filter, object? filterCriteria) => throw new NotSupportedException(SR.NotSupported_SignatureType);
+
         public sealed override InterfaceMapping GetInterfaceMap(Type interfaceType) => throw new NotSupportedException(SR.NotSupported_SignatureType);
```

Comment:
What about `GetInterfaceMap`? Does this need any annotations?

Reply:
> Maybe we need to annotate it with PublicMethods

Does `PublicMethods` include explicitly implemented methods?

Reply:
Might have to be all the methods on the interface: https://github.com/dotnet/runtime/issues/46912#issuecomment-837179543